nav2_costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -90,38 +102,41 @@ void ObstacleLayer::onInitialize()
 
   global_frame_ = layered_costmap_->getGlobalFrameID();
 
-#if 0
-  double transform_tolerance;
-  node_->get_parameter_or("transform_tolerance", transform_tolerance, 0.2);
-
-  // get the topics that we'll subscribe to from the parameter server
-  std::string topics_string;
-  node_->get_parameter_or("observation_sources", topics_string, std::string(""));
-
-  RCLCPP_INFO(node_->get_logger(), "Subscribed to Topics: %s", topics_string.c_str());
-
   // now we need to split the topics based on whitespace which we can use a stringstream for
   std::stringstream ss(topics_string);
-#endif
 
-/*
+  std::cout << ss.str() << std::endl;
```

Comment:
Is this debug code that was supposed to be deleted?

Reply:
Yes, deleted.